pyaedt/circuit.py

```diff
@@ -1551,3 +1552,107 @@ def browse_log_file(self, filepath=None):
         shutil.copy(latest_file, filepath)
         filename = os.path.basename(latest_file)
         return os.path.join(filepath, filename)
+
+    @pyaedt_function_handler()
+    def connect_circuit_models_from_multi_zone_cutout(
+        self, project_connections, edb_zones_dict, ports=None, schematic_units="mm", model_inc=50
+    ):
+        """Connect circuit model from multi-zone clipped project.
+
+        Parameters
+        ----------
+        project_connections : dic[str][str]
+            project connections dictionarry return from command edb.get_connected_ports_from_multizone_cutout():
```

Comment:
```suggestion
            Dictionary of project connections returned from the
            ``edb.get_connected_ports_from_multizone_cutout()`` method.       
```
We should probably use the :func: direction to make the method a link. This is true for the other descriptions as well.

Reply:
ok thanks